app/helpers/physical_server_helper/textual_summary.rb

```diff
@@ -0,0 +1,65 @@
+module PhysicalServerHelper::TextualSummary
+  def textual_group_properties
+    %i(name model product_name manufacturer machine_type serial_number ems_ref memory cores)
+  end
+
+  def textual_group_relationships
+    %i(host)
+  end
+
+  def textual_group_compliance
+  end
+
+  def textual_group_firmware_details
```

Comment:
Stephen I think how the groups are created has changed now.  There are examples in the PR:  https://github.com/ManageIQ/manageiq-ui-classic/pull/274. Can you check to see if you are in sync with those changes?